stacks/dev/stage-1.yml

```diff
@@ -68,7 +68,7 @@ services:
           memory: 300M
   # Proxy
   traefik:
-    image: traefik
+    image: traefik:v1.7
```

Comment:
This change doesn't seem to be related to the commit bump. 
Why do you need to fix the traefik version?

Reply:
I've created another commit related to this change: _51dd37a1267d517e13f04169ce7d6c7492cd3ccd_ .